addons/im_livechat/__manifest__.py

```diff
@@ -70,7 +70,20 @@
             'im_livechat/static/src/components/messaging_menu/messaging_menu_tests.js',
             'im_livechat/static/src/components/thread_icon/thread_icon_tests.js',
             'im_livechat/static/src/components/thread_textual_typing_status/thread_textual_typing_status_tests.js',
-            'im_livechat/static/src/legacy/public_livechat.js',
+            'im_livechat/static/src/legacy/public_livechat/abstract_message.js',
+            'im_livechat/static/src/legacy/public_livechat/abstract_thread_window.js',
+            'im_livechat/static/src/legacy/public_livechat/abstract_thread.js',
+            'im_livechat/static/src/legacy/public_livechat/cc_throttle_function_object.js',
+            'im_livechat/static/src/legacy/public_livechat/cc_throttle_function.js',
+            'im_livechat/static/src/legacy/public_livechat/document_viewer.js',
+            'im_livechat/static/src/legacy/public_livechat/im_livechat.js',
+            'im_livechat/static/src/legacy/public_livechat/thread_typing_mixin.js',
+            'im_livechat/static/src/legacy/public_livechat/thread.js',
+            'im_livechat/static/src/legacy/public_livechat/timer.js',
+            'im_livechat/static/src/legacy/public_livechat/timers.js',
+            'im_livechat/static/src/legacy/public_livechat/website_livechat_message.js',
+            'im_livechat/static/src/legacy/public_livechat/website_livechat_window.js',
+            'im_livechat/static/src/legacy/public_livechat/website_livechat.js',
```

Comment:
As per discussed, revert change in legacy file. No big deal if they are not converted to new modules since we plan to rewrite them soon enough. It's not worth losing the history of the files in git.

Reply:
@seb-odoo fixed